src/v2/cookbook/creating-custom-scroll-directives.md

```diff
@@ -36,7 +36,7 @@ new Vue({
   }
 })
 ```
-
+When the element is inserted, a function, `f`, is created with the event, `evt`, as it's parameter. Then the scroll event listener is added, with the function `f`. When the user scrolls the `handleScroll` function checks if the `scrollY` of the window is greater than 50. If so, the element is displayed and the event listener is removed. 
```

Comment:
I don't think we should explain the code. Good code is self-explanatory, and what we have here is not an exception.